_harp/public/pages/html.md

```diff
@@ -1,12 +1,83 @@
-## Philosophy
-...
+## Introduction
+
+Let's face it, the web is built with HTML and we write a ton of it 👨‍💻. So let's aim write it well! Besides, who wants to wade through a sea of `<div>`s all day!
 
 ## Semantics
-...
+When writing HTML markup, semantics are very, very important. Semantics actually give meaning to your markup which helps with readability, maintainability, accessibility and other benefits. An example of semantic markup would be using a `<nav>` element for a menu instead of a `<div>` tag, or an `<article>` tag for a blog post. Example of good semantics:
```

Comment:
Elements should be used for their intended semantic purpose.

Reply:
love it!